src/scripts/utils.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import os
+from constants import _constants
+
+def _write_file(settings, content):
```

Comment:
The _ prefix is not needed, since it's an util function.

Reply:
I removed it